### PR TITLE
[New Package] minecraft

### PR DESCRIPTION
**Pull request by saltedcoffii**

### CODE REVIEW

File: packages/minecraft.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
+require 'package'
+
+class Minecraft < Package
+  description 'A sandbox video game developed by Mojang Studios AB.'
+  homepage 'https://minecraft.net'
+  version '**filler**' # Not sure what to put here since its the launcher version not the game version.
```
Comment (uberhacker) [inline packages/minecraft.rb:6]: 2.2.1867?  Run `alien -tc Minecraft.deb` to find the version.

Comment (saltedcoffii) [reply packages/minecraft.rb:6]: 2.2.1867

Comment (saltedcoffii) [reply packages/minecraft.rb:6]: alien -tc wasn't working so i did
```shell
ar x Minecraft.deb
tar xf control.tar.gz
cat control
```

Comment (uberhacker) [reply packages/minecraft.rb]: The resulting file after `alien -tc ...` will have the version in the filename.

Comment (saltedcoffii) [reply packages/minecraft.rb]: ohhh i gotcha.

**Pull request merged**